app/views/home/index.html.erb

```diff
@@ -6,12 +6,40 @@
   </p>
 </div>
 
-<h2><%= t('latest_projects') %></h2>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-4">
+      <h2><%= t('latest_projects') %></h2>
 
-<ul>
-  <% cache do %>
-    <% @latest_projects.each do |project| %>
-      <li><%= link_to(project.name, project_path(project.id)) %></li>
-    <% end %>
-  <% end %>
-</ul>
+      <ul>
+        <% cache do %>
```

Comment:
Please have a look at the current code: https://github.com/mezuro/prezento/blob/master/app/views/home/index.html.erb#L12

It already works like this and get expired from time by time. The responsible for choosing the expiration time is the engine (memcached).

Maybe explicit expirations would be easier for developers to understand the code. If this is your point what do you think about an enhancement issue?